widget/hyperlink_test.go

```diff
@@ -35,19 +41,22 @@ func TestHyperlink_SetText(t *testing.T) {
 }
 
 func TestHyperlink_SetUrl(t *testing.T) {
-	var sUrl string = "url1"
+	sUrl, err := url.Parse("https://github.com/fyne-io/fyne")
+	assert.Nil(t, err)
 
 	// test constructor
 	hyperlink := NewHyperlink("Test", sUrl)
 	assert.Equal(t, sUrl, hyperlink.Url)
 
 	// test setting functions
-	sUrl = "https://fyne.io"
+	sUrl, err = url.Parse("https://fyne.io")
+	assert.Nil(t, err)
 	hyperlink.SetUrl(sUrl)
 	assert.Equal(t, sUrl, hyperlink.Url)
-	sUrl = "duck.com"
-	uUrl, err := url.Parse(sUrl)
+	sUrl, err = url.Parse("duck.com")
```

Comment:
Seems like this through line 61 can be taken out, redundant of above test

Reply:
should be fixed by next commit